webapp/channels/src/components/channel_info_rhs/about_area_channel.tsx

```diff
@@ -27,6 +27,18 @@ const ChannelPurpose = styled.div`
     }
 `;
 
+const ChannelDescriptionHeading = styled.div`
+    color: var(--center-channel-color-75, rgba(63, 67, 80, 0.75));
```

Comment:
we dont need this fallback
```
color: rgba(var(xxx...
```

Reply:
We don't have a variable defined by name `--center-channel-color-75`, that's the reason added the fallback.